### ISSUE TITLE
UI review of Barcodes & Containers tile

### ISSUE THREAD

**Issue by mkoo** [ArctosDB/arctos#5749]
Here's where we currently stand:
![Firefox_Screenshot_2023-03-02T20-54-54 192Z](https://user-images.githubusercontent.com/2523089/222551603-331b8861-adec-49c1-b60d-d77a3b64c2a5.png)

rules to consider: alphabetical ordering, no dashes

**Comment by KyndallH** [comment]
Here are two options. They can also be viewed on t[his google sheet](https://docs.google.com/spreadsheets/d/1jGSP4K4NQUcxeUTdKv-li8CqBhURhEoawJMbDTs8XEg/edit?usp=sharing), where others can add/play around with variations. 

<img width="993" alt="Screen Shot 2023-03-02 at 12 42 10 PM" src="https://user-images.githubusercontent.com/16887896/222562684-b9365826-e63f-4e8d-bdcd-c9f5683eb247.png">

**Comment by campmlc** [comment]
I do like the clarity of option 1, except that Find Containers is harder to find. Maybe if we used " Container Find" and drop the "s" for both of those, they'd sort higher up in list?
Also, propose to change Container from Barcode Get to just Container from Barcode.
But I could also go with option 2 if @KyndallH prefers. In this case, actions are useful for sorting, since everything here is already some flavor of Container.

**Comment by dustymc** [comment]
Since there are now spreadsheets involved, I have a request. Don't forget this view: https://arctos.database.museum/directory.cfm?dv=table&access=all&directaccess=all, updating the description where it's needed would be useful.

**Comment by mkoo** [comment]
Thanks for getting spreadsheets involved!
I favor Option 1 too since it's logical and the group-by noun works for me

**Comment by KyndallH** [comment]
Do you have any ideas if we go with nouns first on how to make "Container from Barcode Get" work? That one sounds awful. And "GUID from barcode get"

I agree with removing the "S" to get conatiner find moved up higher. 

<img width="336" alt="Screen Shot 2023-03-03 at 11 03 44 AM" src="https://user-images.githubusercontent.com/16887896/222815779-e3a5a4df-d7ab-4f4a-8930-70478cd03b6b.png">

**Comment by mkoo** [comment]
> Do you have any ideas if we go with nouns first on how to make "Container from Barcode Get" work? That one sounds awful. And "GUID from barcode get"
> 
I think Yoda approves though!

For the rest of us:
How about "Container Find Barcode" or "Container Get Barcode"? or "Container Barcode Lookup"

And similarly "GUID Barcode Lookup"

**Comment by dustymc** [comment]
Effectively communicative GUID Barcode Lookup is not.

**Comment by campmlc** [comment]
Just Container from Barcode - no verb

**Comment by campmlc** [comment]
Guid from Barcode

**Comment by mkoo** [comment]
happy I am

**Comment by campmlc** [comment]
No Yoda emojis on this thing :)

**Comment by mkoo** [comment]
To sum, this is my understanding so far (from [spreadsheet](https://docs.google.com/spreadsheets/d/1jGSP4K4NQUcxeUTdKv-li8CqBhURhEoawJMbDTs8XEg/edit#gid=0))

Suggestion 1 (noun first)
--
Barcode Series Reserve
Barcode Swap Bulk
Barcode Swap Tool
Container Add Parts
Container Bulkload
Container Edit Bulk
Container Environment Bulkload
Container Find
Container Find Mismatch
Container from Barcode
Container Move
Container Move Bulk
Container Position Create Bulk
GUID from Barcode
----

To avoid the yoda-speak, can we revist the no-dash rule? How about colons-- it would make a lot more sense (and work in other tiles too)

So for example:

Suggestion 3 (use punctuation!)
--
Barcode: Series Reserve
Barcode: Swap Bulk
Barcode: Swap Tool
Container: Add Parts
Container: Bulkload
Container: Edit Bulk
Container: Environment Bulkload
Container: Find
Container: Find Mismatch
Container from Barcode
Container: Move
Container: Move Bulk
Container: Create Position Bulk
GUID from Barcode

**Comment by campmlc** [comment]
I like those suggestions, but would add the following:
Change " Barcode Swap Tool" to just "Barcode Swap" and Barcode Swap Bulk "
so matches the Move and Move Bulk options below. Fewer words, sorts better.
Also, shouldn't Container from Barcode sort just above Guid from Barcode,
since both lack colons? It looks out of order.

**Comment by mkoo** [comment]
It orders if you reorder that column! 

How 'bout:


Suggestion 3 (use punctuation!)
--
Barcode: Series Reserve
Barcode: Swap
Barcode: Swap by Bulk
Container from Barcode
Container: Add Parts
Container: Bulkload
Container: Create Position Bulk
Container: Edit Bulk
Container: Environment Bulkload
Container: Find
Container: Find Mismatch
Container: Move
Container: Move by Bulk
GUID from Barcode

**Comment by mkoo** [comment]
I like the Suggestion 4!

**Comment by Jegelewicz** [comment]
See Suggestion 5 for addition of request here - https://github.com/ArctosDB/arctos/issues/5769#issuecomment-1461211611

**Comment by Jegelewicz** [comment]
For Consideration: Should the Titles of the various Arctos pages match whatever we put in the tiles?

Also - can we get a consensus? Do we need a UI meeting?

**Comment by mkoo** [comment]
To sum, this is my understanding so far (from [spreadsheet](https://docs.google.com/spreadsheets/d/1jGSP4K4NQUcxeUTdKv-li8CqBhURhEoawJMbDTs8XEg/edit#gid=0))

Suggestion 1 (noun first)
--
Barcode Series Reserve
Barcode Swap Bulk
Barcode Swap Tool
Container Add Parts
Container Bulkload
Container Edit Bulk
Container Environment Bulkload
Container Find
Container Find Mismatch
Container from Barcode
Container Move
Container Move Bulk
Container Position Create Bulk
GUID from Barcode
----

To avoid the yoda-speak, can we revist the no-dash rule? How about colons-- it would make a lot more sense (and work in other tiles too)

So for example:

Suggestion 3 (use punctuation!)
--
Barcode: Series Reserve
Barcode: Swap Bulk
Barcode: Swap Tool
Container: Add Parts
Container: Bulkload
Container: Edit Bulk
Container: Environment Bulkload
Container: Find
Container: Find Mismatch
Container from Barcode
Container: Move
Container: Move Bulk
Container: Create Position Bulk
GUID from Barcode

Yes, I'll put on the AWG agenda but maybe a UI meeting next week so we can agree on at least a few tiles

**Comment by campmlc** [comment]
Colons work!

**Comment by KyndallH** [comment]
Suggestion 5 is fine! I'm okay with moving forward with it!

**Comment by campmlc** [comment]
I have to admit that I really prefer Suggestion 3, if we can go back to that. It is closer to the familiar terms we already use, and the language is simpler. It makes a big difference visually if we can reduce the number of words and make the words shorter, which that version does. Suggestion 5 makes everything look the same and adds extra syllables and words, so I have to read and think about each choice, which made my brain hurt.  I guess I just rely a lot on the visual appearance of each category to find what I'm looking for.

**Comment by KyndallH** [comment]
I'm a short-timer now so I'm fine with 3. I'm pretty much fine with anything but the current standard mess.

**Comment by mkoo** [comment]
going with Option 3 with some tweaks...

>short-timer
this makes me sad! 😢

**Comment by Jegelewicz** [comment]
Changes have been made - if @campmlc and @KyndallH approve, they can close this.

**Comment by mkoo** [comment]
looks like all changes have occurred from the UI committee and Issues meeting-- thanks, teresa
